docs/source/modin/supported/series_supported.rst

```diff
@@ -76,11 +78,11 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``add_suffix``              | Y                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``agg``                     | P                               |                                  | ``Y`` when  function is one of `count`, `mean`,    |
-|                             |                                 |                                  | `min`, `max`, `sum`, `median`; `std` and `var`     |
-|                             |                                 |                                  | supported with `ddof=0` or `ddof=1`; `quantile` is |
-|                             |                                 |                                  | supported when `q` is the default value or a       |
-|                             |                                 |                                  | scalar                                             |
+| ``agg``                     | P                               |                                  | ``Y`` when  function is one of ``count``,          |
+|                             |                                 |                                  | ``mean``, ``min``, ``max``, ``sum``, ``median``;   |
+|                             |                                 |                                  | ``std`` and ``var`` supported with ``ddof=0`` or   |
```

Comment:
```suggestion
|                             |                                 |                                  | ``std``, or ``var`` supported with ``ddof=0`` or   |
```

Reply:
This isn't a continuation of the list, ``std`` and ``var`` are only supported with ``ddof=0`` (that's why they're separated from the other aggregations by the semicolon).